repos/fdbt-site/src/pages/api/manageOperatorDetails.ts

```diff
@@ -28,9 +28,10 @@ export const collectErrors = (operatorDetails: OperatorDetails): ErrorInfo[] =>
     Object.entries(operatorDetails)
         .filter((entry) => {
             const inputValue = entry[1];
+            const inputKey = entry[0];
 
-            const entryHasInvalidCharacters = invalidCharactersArePresent(inputValue);
-
+            const entryHasInvalidCharacters =
+                inputKey === 'url' ? invalidUrlInput(inputValue) : invalidCharactersArePresent(inputValue);
```

Comment:
I think we should be leaving this chunk of code as it was, and using 'invalidUrlInput' function on line 56 instead.

Reply:
we have to have something to we stop  passing the url key to the `nvalidCharactersArePresent` function.